src/Models/CategoriesModel.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+require_once 'src/Entities/Categories.php';
+
+Class CategoriesModel{
```

Comment:
Class should be class (little c) and the opening { should be on the next line please

Reply:
Generally model names are singular, so this should be CategoryModel

Reply:
All changed!.